src/api/get.js

```diff
@@ -1,16 +1,12 @@
 import Wreck from '@hapi/wreck'
-import { config } from '../config/config.js'
-import { createLogger } from '../common/helpers/logging/logger.js'
-
-const logger = createLogger()
+import { buildBackendUrl } from './build-backend-url.js'
+import { handleBackendRequest } from './handle-backend-request.js'
 
 export async function get (url) {
```

Comment:
Pedentic point.

Can we call the param `path` as the function is building the url with the path being appended from the param